cmd/gcp-controller-manager/csr_approver.go

```diff
@@ -311,7 +311,7 @@ func isNodeClientCert(csr *capi.CertificateSigningRequest, x509cr *x509.Certific
 	if !isNodeCert(csr, x509cr) {
 		return false
 	}
-	if len(x509cr.DNSNames) > 0 || len(x509cr.IPAddresses) > 0 {
+	if len(x509cr.DNSNames) > 0 || len(x509cr.EmailAddresses) > 0 || len(x509cr.IPAddresses) > 0 || len(x509cr.URIs) > 0 {
```

Comment:
`EmailAddresses` is already filtered out in `isNodeCert`.

Reply:
Good catch, fixed.